.github/ISSUE_TEMPLATE/question-issue.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Ask a question about or report an issue with GEOS-Chem
```

Comment:
Same comments as for the new feature and discussion template, except more checkboxes.